src/appkit.cs

```diff
@@ -16942,7 +16947,7 @@ public partial interface NSTextInputContext {
 		NSTextInputContext CurrentInputContext { get; }
 
 		[Export ("client")]
-		NSTextInputClient Client { get; }
+		INSTextInputClient Client { get; }
```

Comment:
My mistake. They were added in the xcode8 work, but not 10.12 specific. In any case, we can change since they are new.